proto/ibc/core/channel/v1/channel.proto

```diff
@@ -199,3 +199,9 @@ message Timeout {
   // block timestamp (in nanoseconds) after which the packet or upgrade times out
   uint64 timestamp = 2;
 }
+
+// Params defines the set of IBC channel parameters.
+message Params {
+  // timeout after which channel upgrades will time out.
```

Comment:
if we do the the calc of adding timeout height to current block height as well as adding timeout timestamp to current block time then we should document it here that's its basically treated as a relative timeout (we'd want to document this in the our markdown docs as well I guess)